ListDetail/src/main/java/com/microsoft/device/display/samples/listdetail/ui/view/MainPage.kt

```diff
@@ -7,19 +7,25 @@ package com.microsoft.device.display.samples.listdetail.ui.view
 
 import androidx.compose.foundation.Image
 import androidx.compose.runtime.Composable
+import androidx.compose.runtime.getValue
+import androidx.compose.runtime.mutableStateOf
+import androidx.compose.runtime.saveable.rememberSaveable
+import androidx.compose.runtime.setValue
 import androidx.compose.ui.Modifier
 import androidx.compose.ui.res.painterResource
-import com.microsoft.device.display.samples.listdetail.models.AppStateViewModel
 import com.microsoft.device.dualscreen.twopanelayout.TwoPaneLayout
 import com.microsoft.device.dualscreen.twopanelayout.TwoPaneMode
 import com.microsoft.device.dualscreen.windowstate.WindowState
 
 @Composable
-fun ListDetailApp(viewModel: AppStateViewModel, windowState: WindowState) {
+fun ListDetailApp(windowState: WindowState) {
+    var selectedIndex by rememberSaveable { mutableStateOf(0) }
+    val updateSelectedIndex: (Int) -> Unit = { newIndex -> selectedIndex = newIndex }
+
     TwoPaneLayout(
         paneMode = TwoPaneMode.HorizontalSingle,
-        pane1 = { ListViewWithTopBar(viewModel) },
-        pane2 = { DetailViewWithTopBar(windowState.isDualPortrait(), viewModel) }
+        pane1 = { ListViewWithTopBar(windowState.isDualPortrait(), selectedIndex, updateSelectedIndex) },
+        pane2 = { DetailViewWithTopBar(windowState.isDualPortrait(), selectedIndex) }
     )
 }
 
```

Comment:
Would it be possible to move the `ImageView` composable to its own file? I think it would fit better either under a new ui.components package or as a new file in the ui.view package.

Also, I think it would be good to add a parameter for the `contentDescription` field so it's not null!